src/starkbank/settings.clj

```diff
@@ -7,7 +7,7 @@
 (defn user
   "sets a default user (project) to be automatically used in all requests"
```

Comment:
"sets a default user (project) to be automatically used in all requests" -> "sets a default user (project or organization) to be automatically used in all requests"